### ISSUE TITLE
Pack qty and pack size values swap when adding/editing item in Inbound Shipment

### ISSUE THREAD

**Issue by noel-yeldos** [msupply-foundation/open-msupply#5761]
## What went wrong? 😲
Issue seems to be a development issue. Checked in demo server using 2.4.01 and bus is NOT present there.

When adding a new item in inbound shipment default pack size is set correctly e.g 100. However, when setting pack qty to 1 and then then saving, pack size becomes 1 and pack qty becomes 100. Same also happens when editing items added from master list. 

https://github.com/user-attachments/assets/0da736fc-6790-479a-81bb-2c746ac09745

## Expected behaviour 🤔
Pack size and pack qty values should not swap when adding or editing an item in inbound shipment.

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to Replenishment -> Inbound shipment
2. Click on existing shipment or create a new one
3. Add a new item
4. See default pack size is being set correctly
5. Enter pack qty 1 and click ok
6. See that values are swapped in list view
7. Add items from master list
8. See that default pack sizes are set correctly
9. Edit item and add pack qty 1
10. See values swapped after save

## Your environment 🌱

- Open mSupply Version:2.5.00
- Legacy mSupply Central Server Version: 7.18.20

- Platform: Chrome browser Mac

- Database type:

**Comment by andreievg** [comment]
@noel-yeldos do you have pack to one preference set for the store ? 

User should not be confused though even if it's set.

**Comment by noel-yeldos** [comment]
@andreievg ah I didn't even know that was a preference! So technically if pack to one is set for the store, should the modal when the item is initially selected default to 1 (Currently it is showing item default pack size)? Or do we need some sort of indicator that pack to one is set, and the pack size will change to 1 when you add the item? 
![Screenshot 2024-12-13 at 10 49 30 AM](https://github.com/user-attachments/assets/0d090736-1133-4bb2-86df-081e26a04908)

@lache-melvin I know you had mentioned adding a UI pop-up here or disabling editing of pack size

**Comment by roxy-dao** [comment]
There's an issue to display store preferences somewhere... The user shouldn't be confused if they have the store preference set for their store 🤔 Should be all covered in training me thinks

**Comment by lache-melvin** [comment]
Fair enough - still think at least having pack size field show `1` and be disabled would make more sense, but we've probably got bigger fish in the backlog 😁

**Comment by andreievg** [comment]
> Fair enough - still think at least having pack size field show 1 and be disabled would make more sense

Triage, when creating inbound shipment, and the supplier is using other then 1 pack size, it's a lot easier for user to enter supplier version of the inbound shipment (cost/sell price conversions etc..)

**Comment by andreievg** [comment]
Triage, user should be able to enter whatever packsize they want, they will be notified that pack size will be converted to 1, in a modal inline message. Same for stocktake and inventory adjustment
